### ISSUE TITLE
"export 'attachLinkToDocumentation' was not found in '@ckeditor/ckeditor5-utils/src/ckeditorerror'

### ISSUE THREAD

**Issue by idonans** [Technologie-Visao/ckeditor5-video#7]
```
 WARNING  Compiled with 1 warnings                                                                                                                                                                                                                                       3:30:02 ├F10: PM┤
 warning  in ./node_modules/@visao/ckeditor5-video/src/videostyle/utils.js

"export 'attachLinkToDocumentation' was not found in '@ckeditor/ckeditor5-utils/src/ckeditorerror'
```

**Comment by alexdaube** [comment]
This seems have been removed with ck-editor 5 v23.1.0 release. I will upgrading this package to support 23.1.0 very shortly.

**Comment by alexdaube** [comment]
Will be fixed with #2
